sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/src/Azure.AI.TextAnalytics.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>This is the Microsoft Azure Cognitive Services Text Analystics Service client library</Description>
+    <AssemblyTitle>Microsoft Azure.AI.TextAnalytics client library</AssemblyTitle>
+    <Version>1.0.0-preview.1</Version>
+    <PackageTags>Microsoft Azure Text Analytics</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+      Initial Preview Release.
+      ]]>
+    </PackageReleaseNotes>
+    <GenerateAPIListing>false</GenerateAPIListing>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>$(NoWarn);3021</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
The guidance is to prefer Span<> over unsafe pointers and try to avoid usafe blocks as much as possible

Reply:
I noticed we have this on our KV projects as well but don't make use of it. Maybe copy and paste issue? I don't recall seeing you depend on it anywhere, either.

Reply:
I'd recommend how we did it for PowerShell. All projects simply reference the same targets/props file, and then you can have special cases in the project files. Otherwise, standard settings are in the same place.
Example, where this project needed AllowUnsafeBlocks but that setting isn't there for almost all other projects:
https://github.com/Azure/azure-powershell/blob/master/src/KeyVault/KeyVault/KeyVault.csproj

Then the shared file:
https://github.com/Azure/azure-powershell/blob/master/src/Az.props

Reply:
We have common props file: https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Directory.Build.Data.props